### PR TITLE
Add find primitive root

### PR DESCRIPTION
**Pull request by D-Sinus**
(Put an `X` inside the `[ ]` to denote check mark `[X]`.)

- [X] **If creating a new file :**
  - [X] added links to it in the README files ?
  - [X] included tests with it ?
  - [X] added description (overview of algorithm, time and space complexity, and possible edge case) in docstrings ?

- [ ] **if done some changes :**
  - [ ] wrote short description in the PR explaining what the changes do ?
  - [ ] Fixes #[issue number] if related to any issue

- [ ] **other**

### CODE REVIEW

**Comment by D-Sinus** [comment]
To find primitive root, Euler's totient function is used.
At first I think I need to implement that function by myself,
but 'goswami-rahul' already implemented that function.
So I copy his/her code and write comment about that (this is not my code)
Would it be allowed? If not, I'll delete that part and re-implement that function by myself.
Please let me know. Thanks.

File: algorithms/maths/find_primitive_root_simple.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+"""
+
+def find_primitive_root(n):
+    if (n == 1):
+        return [0]
+        """ Exception Handeling :
+        0 is the only primitive root of 1 """
+    else:
+        phi = euler_totient(n)
+        pRootList = []
```
Comment (keon) [inline algorithms/maths/find_primitive_root_simple.py]: please use consistent naming convention

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
 
-- [ ] **If creating a new file :**
-  - [ ] added links to it in the README files ?
-  - [ ] included tests with it ?
-  - [ ] added description (overview of algorithm, time and space complexity, and possible edge case) in docstrings ?
+- [X] **If creating a new file :**
+  - [X] added links to it in the README files ?
+  - [X] included tests with it ?
+  - [X] added description (overview of algorithm, time and space complexity, and possible edge case) in docstrings ?
```
Comment (keon) [inline PULL_REQUEST_TEMPLATE.md]: no need to edit the template

File: algorithms/maths/find_primitive_root_simple.py (hunk 1 above)
Comment (D-Sinus) [reply algorithms/maths/find_primitive_root_simple.py]: Change pRootList  into p_root_list.

File: PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (D-Sinus) [reply PULL_REQUEST_TEMPLATE.md]: [Edit] Revert PULL_REQUEST_TEMPLATE.md
